rsts/community/roadmap.rst

```diff
@@ -50,18 +50,18 @@ Issues by Theme
-| scale       | These issues deal with performance,  reliability and           | `Scale issues <https://github.com/flyteorg/flyte/labels/scale>`_                      | We are always working on these issues and we would love to head feedback about what you                     |
-|             | scalability of Flyte                                           |                                                                                       | would want to change or what should we prioritize                                                           |
+| Scale       | These issues deal with performance,  reliability and           | `Scale issues <https://github.com/flyteorg/flyte/labels/scale>`_                      | We are always working on these issues and we would love to hear feedback about what you                     |
+|             | scalability of Flyte                                           |                                                                                       | would want to change or what we should prioritize.                                                           |
 +-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
-| Contribute  | If you are looking to contribute and want a great first issue, | `Contribute issues <https://github.com/flyteorg/flyte/labels/good%20first%20issue>`_  | These are the best issues to get started with                                                               |
-|             | look at these issues                                           |                                                                                       |                                                                                                             |
+| Contribute  | If you are looking to contribute and want a great first issue, | `Contribute issues <https://github.com/flyteorg/flyte/labels/good%20first%20issue>`_  | These are the best issues to get started with.                                                               |
+|             | check out these issues                                           |                                                                                       |                                                                                                             |
 +-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
 
```

Comment:
+-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
| Theme       | Description                                                    | Open Issues                                                                           | Comment                                                                                                     |
+-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
| Bugs        | Currently known and open bugs.                                 | `Bugs <https://github.com/flyteorg/flyte/labels/bug>`_                                | We are always working on bugs. Open a new one `here <https://github.com/flyteorg/flyte/issues/new/choose>`_.|
+-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
| Security    | Issues related to security enhancements.                       | `Security issues <https://github.com/flyteorg/flyte/labels/security>`_                |                                                                                                             |
+-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
| Docs        | All the documentation issues.                                  | `Docs issues <https://github.com/flyteorg/flyte/labels/documentation>`_               |                                                                                                             |
+-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
| Features    | All the new features in development.                           | `Features issues <https://github.com/flyteorg/flyte/labels/enhancement>`_             |                                                                                                             |
+-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
| Plugins     | New capabilities, plugins that are being built into Flyte.     | `Plugin issues <https://github.com/flyteorg/flyte/labels/plugins>`_                   | This is one of the best places to start contributing to Flyte. Issues with both                             |
|             | These could be hosted services, K8s native execution, etc.     |                                                                                       | labels ``plugins`` and ``Flytekit`` refer to purely client-side plugins and are quick to contribute.        |
+-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
| Scale       | These issues deal with performance, reliability, and           | `Scale issues <https://github.com/flyteorg/flyte/labels/scale>`_                      | We have always been working on enhancing the scalability of Flyte. We would love                            |
|             | scalability of Flyte.                                          |                                                                                       | to hear feedback about what you want to change or what we should prioritize.                                |
+-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
| Contribute  | If you are looking to contribute and want a great first issue, | `Contribute issues <https://github.com/flyteorg/flyte/labels/good%20first%20issue>`_  | These are the best issues to get started with.                                                              |
|             | look at these issues.                                          |                                                                                       |                                                                                                             |
+-------------+----------------------------------------------------------------+---------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+